submissions/Myroslava-Kryvoshei/hooli_style_popup/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link type="Image/x-icon" href="/img/favicon.png" rel="icon" />
+    <title>Hooli-Style Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__title">Mail</a></li>
+          <li class="nav__item"><a href="#" class="nav__title">Images</a></li>
+          <li class="nav__item">
+            <input
+              type="checkbox"
+              class="popup__checkbox popup__checked"
+              id="popup-checkbox"
+            />
+            <label for="popup-checkbox" class="label__popup">
+              <img
+                src="img/popup-button.png"
+                alt="Popup icon"
+                class="popup__icon"
+              />
+            </label>
```

Comment:
Your popup should be accessible from keyboard. [Watch video](https://share.vidyard.com/watch/QkedJ8r8naHqWUQZZ33ixH?)